pyproject.toml

```diff
@@ -0,0 +1,18 @@
+# As per https://github.com/pypa/setuptools/blob/main/docs/userguide/quickstart.rst
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "hca-import-validation"
+version = "0.0.7"
+requires-python = ">=3.6"
+dependencies = [
```

Comment:
ish - I misunderstood the setuptools doc and created the toml but left the setup.cfg, which I've now sorted out and ported all of the setup.cfg metadata to the pyproject.toml. 
Requirements.txt is separate as that pins the versions of the dependencies for install. This is the common practice, rather than doing so in the build dependencies. This also allows for local envs to setup via  pip install -r requirements.txt.